cabal.project

```diff
@@ -43,3 +43,9 @@ write-ghc-environment-files: always
 -- IMPORTANT
 -- Do NOT add more source-repository-package stanzas here unless they are strictly
 -- temporary! Please read the section in CONTRIBUTING about updating dependencies.
+
+source-repository-package
+  type: git
+  location: https://github.com/input-output-hk/cardano-api
+  tag: c08769a19d39c28bd3edb6885247201ea34d7949
+  subdir: cardano-api
```

Comment:
Remove this when `cardano-api-8.26.0.0` is ready: https://github.com/input-output-hk/cardano-api/pull/326